FinSearchUnified/Tests/Helper/UrlBuilderTest.php

```diff
@@ -142,6 +142,28 @@ public function testHandlesUnknownClientIp()
         $this->assertEquals('UNKNOWN', $usedIpInRequest);
     }
 
+    /**
+     * @throws \Exception
+     */
+    public function testOutputAdapterParameterInUrl()
+    {
+        $urlBuilder = new UrlBuilder($this->httpClient);
+
+        $criteria = new Criteria();
+        $criteria->offset(0)->limit(2);
+
+        $urlBuilder->buildQueryUrlAndGetResponse($criteria);
+        /** @var Zend_Uri_Http $requestedUrl */
+        $requestedUrl = $this->httpClient->getUri();
+        $path = $requestedUrl->getPath();
+        $this->assertNotContains('xml_2.0', $path, 'Expected "xml_2.0" to not be passed in requested URL ');
+
+        $queryArray = $requestedUrl->getQueryAsArray();
+
+        $this->assertArrayHasKey('outputAdapter', $queryArray, 'outputAdapter was not found in query parameters');
```

Comment:
Add double quotes to `outputAdapter` and `XML_2.0` e.g.:
* `'"outputAdapter" was not found ...'`
* `'"XML_2.0" was not found ...'`